htdocs/PI/write/PIWriteRequest.php

```diff
@@ -569,6 +569,17 @@ private function checkUserAuthenticated () {
       }
     }
 
+    private function checkAuthorisation (Site $siteService, \Site $site, $identifier, $indentifierType) {
```

Comment:
Whhy does the second `Site` have a `\`, but the first doesn't?

Reply:
https://www.php.net/manual/en/language.namespaces.php Sorry

Reply:
at the hackathon: 
> George: "it's to do with namespaces, the first Site object is actually a SiteService and we are in the right namespace for that, the second Site is an actual Site object, and we aren't in the  namespace for that..."
> Greg: *wails in pain*